packages/financial-templates-lib/src/clients/InsuredBridgeL1Client.ts

```diff
@@ -296,7 +296,7 @@ export class InsuredBridgeL1Client {
           proposerBond: depositRelayedEvent.returnValues.relay.proposerBond,
           finalFee: depositRelayedEvent.returnValues.relay.finalFee,
           settleable: SettleableRelay.CannotSettle,
-          blockTime: Number((await this.l1Web3.eth.getBlock(depositRelayedEvent.blockNumber)).timestamp),
+          blockNumber: depositRelayedEvent.blockNumber,
```

Comment:
One minor note: I agree that it's probably good to avoid doing node calls for each relay event in the window. However, the slowness (I think) is because we were doing this call in series rather than doing it inside a Promise.all. For instance, we could probably speed the bot up more by putting the outer loop in a Promise.all.